### ISSUE TITLE
its working can bet?

### ISSUE THREAD

**Issue by lordmen99** [Oznerol92/prediction-bot#1]
Hi, its can bet in live round?

**Comment by Oznerol92** [comment]
Hi I think it doesn't work 100%, I was testing it but then I started to work on other stuff.

Would you like to work on it?

**Comment by lordmen99** [comment]
how show log/debug. you mean working can make prediction and bet?

**Comment by Oznerol92** [comment]
Ok I've look at it and it is connected with the smart contract so after you connect metamask and start the bot it starts to log in browser console, but it's not placing any bet yet.

Let me know if you get it going or you need any help

**Comment by lordmen99** [comment]
ok, when bet its auto or need confirm ?

**Comment by Oznerol92** [comment]
it doesn't bet yet, it's commented out in https://github.com/Oznerol92/prediction-bot/blob/897b8bbdfd4b049ad86a88876bc1131691bf3de8/public/src/Components/PredictionBot.jsx#L920

but the strategy is not written yet I'm pretty sure, I have only a simulation of martingale with some custom parameters and you can see it here https://github.com/Oznerol92/prediction-bot/blob/master/utils/martingale.js

**Comment by lordmen99** [comment]
yes, i see that line. but when try test bet, its need confirmation?

**Comment by Oznerol92** [comment]
no I think it's only doing it as a simulation you don't need to confirm transaction

**Comment by lordmen99** [comment]
can u add, log on page / screen?

**Comment by Oznerol92** [comment]
there's an easy frontend if you run `npm run dev`, I'm sorry but I don't have a lot of time right now to follow this project as I'd like to. Are you willing to do it?

**Comment by lordmen99** [comment]
i try uncomment this startCheckingRounds, but page blank

**Comment by Oznerol92** [comment]
yeah it's full of bugs. I'd suggest go through the code first to see what it does before running it

**Comment by Oznerol92** [comment]
> yeah it's full of bugs. I'd suggest go through the code first to see what it does before running it

another thing. If you just want to follow the epoch you have to `npm run dev` and then connect metamask. Then you can start the bot and you can see in the console that is checking the epoch.

The function is this one https://github.com/Oznerol92/prediction-bot/blob/897b8bbdfd4b049ad86a88876bc1131691bf3de8/public/src/Components/PredictionBot.jsx#L86-L107

You can uncomment or add your stuff here and it will run on every change of epoch, or at least it should work like that

**Comment by lordmen99** [comment]
i wll try fix it, and make can bet auto. :)

**Comment by Oznerol92** [comment]
That's great! Let me know if you need anything

**Comment by Oznerol92** [comment]
> i wll try fix it, and make can bet auto. :)

Hey any luck?
